doc/source/oss-users.rst

```diff
@@ -0,0 +1,80 @@
+.. _ray-oss-list:
+
+Open source projects that run on Ray
+====================================
+
+The Ray ecosystem is rapidly growing! This page lists projects that are known to use Ray. This does not constitute an endorsement.
+
+*Using Ray?* `Fill out this form <https://forms.gle/ps4S1C955kjotxd57>`__ *and we'll be in contact to get added to the list.*
+
+Horovod
+-------
+
+Horovod is a distributed deep learning training framework for TensorFlow, Keras, PyTorch, and Apache MXNet. The goal of Horovod is to make distributed deep learning fast and easy to use.
+
+Github: `https://github.com/horovod/horovod/ <https://github.com/horovod/horovod/>`_
+
+MARS
+----
+
+Mars is a tensor-based unified framework for large-scale data computation which scales Numpy, Pandas and Scikit-learn.
+
+Github: `https://github.com/mars-project/mars/ <https://github.com/mars-project/mars/>`_
+
+Spacy
+-----
+spaCy is a library for advanced Natural Language Processing in Python and Cython. It's built on the very latest research, and was designed from day one to be used in real products.
+
+Github: `https://github.com/explosion/spaCy/ <https://github.com/explosion/spaCy/>`_
```

Comment:
Can we link to their documentation on Ray integration, here and elsewhere? The generic project link isn't that useful, it's easy enough to find by googling.

Reply:
I've added integration docs for those that make sense.